popup.js

```diff
@@ -0,0 +1,251 @@
+var axios = require("axios");
```

Comment:
Ok thanks, reading up on this. I need to pay more attention to code organization